src/mono/mono/mini/interp/interp.c

```diff
@@ -9039,6 +9039,10 @@ mono_ee_interp_init (const char *opts)
 	set_context (NULL);
 
 	interp_parse_options (opts);
```

Comment:
Is the intent to append options passed to mono_ee_inter_init (normally from command line --interp= arg) with options passed in env or should env override command line options?